CHANGELOG.md

```diff
@@ -2,29 +2,36 @@
 
 ### Devop
 
-- Fix sentry [#2858](https://github.com/MyEtherWallet/MyEtherWallet/pull/2858)
-- Fix wallet layouts [#2848](https://github.com/MyEtherWallet/MyEtherWallet/pull/2848)
-- add commitlint and husky for branch names [#2845](https://github.com/MyEtherWallet/MyEtherWallet/pull/2845)
-- Send TX changes [#2797](https://github.com/MyEtherWallet/MyEtherWallet/pull/2797)
-- Vuex changes [#2787](https://github.com/MyEtherWallet/MyEtherWallet/pull/2787)
-- Add SWAP feature [#2774](https://github.com/MyEtherWallet/MyEtherWallet/pull/2774)
-- Add decision tree [#2720](https://github.com/MyEtherWallet/MyEtherWallet/pull/2720)
-- Add Privacy and Terms pages [#2733](https://github.com/MyEtherWallet/MyEtherWallet/pull/2733)
-- update mobile wallet menu [#2731](https://github.com/MyEtherWallet/MyEtherWallet/pull/2731)
-- update mobile front menu [#2726](https://github.com/MyEtherWallet/MyEtherWallet/pull/2726)
-- Update team page [#2743](https://github.com/MyEtherWallet/MyEtherWallet/pull/2743)
-- Add Press Kit page [#2742](https://github.com/MyEtherWallet/MyEtherWallet/pull/2742)
-- Fix mew info card UI issues [#2737](https://github.com/MyEtherWallet/MyEtherWallet/pull/2737)
-- Fix how it works UI issues [#2736](https://github.com/MyEtherWallet/MyEtherWallet/pull/2736)
-- update mobile dashboard [#2673](https://github.com/MyEtherWallet/MyEtherWallet/pull/2673)
-- update mobile notification [#2671](https://github.com/MyEtherWallet/MyEtherWallet/pull/2671)
-- update mobile side menu [#2670](https://github.com/MyEtherWallet/MyEtherWallet/pull/2670)
-- update mobile tools [#2669](https://github.com/MyEtherWallet/MyEtherWallet/pull/2669)
-- update mobile how it works [#2668](https://github.com/MyEtherWallet/MyEtherWallet/pull/2668)
-- Update mobile access wallet [#2663](https://github.com/MyEtherWallet/MyEtherWallet/pull/2663)
-- Update mobile create wallet [#2661](https://github.com/MyEtherWallet/MyEtherWallet/pull/2661)
-- Finalize routes [#2556](https://github.com/MyEtherWallet/MyEtherWallet/pull/2556)
-- Update/Add landing, create wallet, access wallet pages [#2493](https://github.com/MyEtherWallet/MyEtherWallet/pull/2493)
-- Apply MEW components to dashboard [#2378](https://github.com/MyEtherWallet/MyEtherWallet/pull/2378)
-- Add MY TOKENS VALUE component for Dashboard [#2015](https://github.com/MyEtherWallet/MyEtherWallet/pull/2015)
-- Update my eth balance component chart [#1987](https://github.com/MyEtherWallet/MyEtherWallet/pull/1987)
+<<<<<<< HEAD
```

Comment:
remove missed conflicts

Reply:
fixed in https://github.com/MyEtherWallet/MyEtherWallet/pull/2863/commits/4bcc3b0ea324edcf408e3d08b3da699983905fb4